tensorflow/lite/micro/kernels/Makefile.inc

```diff
@@ -15,6 +15,9 @@
 
 # This file includes kernel test targets only.
 
+INCLUDES += \
+ -I$(MAKEFILE_DIR)/downloads/kissfft
```

Comment:
let's add to this list instead:
https://github.com/tensorflow/tflite-micro/blob/4c2bd5c98ee297fd5f8f89fd5bfe65c74bd02f3b/tensorflow/lite/micro/tools/make/Makefile#L81-L85

Reply:
Done.